catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -285,7 +279,6 @@
                             !org.jboss.*,
                             !javax.ejb.*,
                             !com.tc.*,
-                            !com.terracottatech.*,
                             !org.terracotta.*,
```

Comment:
Yeah that looks strange off the top of my head. Only thing I can think of is some transitive dependency there.